scripts/bundles/plugins/alias-plugin.ts

```diff
@@ -59,6 +59,9 @@ export function aliasPlugin(opts: BuildOptions): Plugin {
       if (id === '@sys-api-node') {
         return join(opts.buildDir, 'sys', 'node', 'index.js');
       }
+      if (id === '@stencil/core/cli') {
```

Comment:
For my own understanding, why do we make changes to the alias plugin here for string->boolean coercion?

Reply:
since we're adding an import of the `BOOLEAN_CLI_ARGS` in the code which is bundled into `testing/` we need to make sure that module can be resolved, I was only testing locally with esbuild but got a rollup build error on CI so went and added this

the esbuild equivalent change is here:

https://github.com/ionic-team/stencil/pull/5672/files#diff-b593e897dd3eb55a2f6dbab40eec549d1bd820ba3dddbd5c6e6340f1c50ef0c2

Reply:
Makes sense - can you update the commit msg with this context before you merge?